sal_interfaces/MTAOS/MTAOS_Events.xml

```diff
@@ -402,40 +315,26 @@
   </SALEvent>
   <SALEvent>
     <Subsystem>MTAOS</Subsystem>
-    <EFDB_Topic>MTAOS_logevent_wepWarning</EFDB_Topic>
-    <Description>Warning of wavefront estimation pipeline.</Description>
+    <EFDB_Topic>MTAOS_logevent_wepDuration</EFDB_Topic>
+    <Description>Duration of time to process the wavefront image data.</Description>
     <item>
-      <EFDB_Name>timestamp</EFDB_Name>
-      <Description>Time at which an event is recorded.</Description>
-      <IDL_Type>double</IDL_Type>
+      <EFDB_Name>calcTime</EFDB_Name>
+      <Description>Average of calculation time.</Description>
+      <IDL_Type>float</IDL_Type>
       <Units>second</Units>
       <Count>1</Count>
     </item>
-    <item>
-      <EFDB_Name>warning</EFDB_Name>
-      <Description>Specific warning type.</Description>
-      <IDL_Type>int</IDL_Type>
-      <Units>unitless</Units>
-      <Count>1</Count>
-    </item>
   </SALEvent>
   <SALEvent>
     <Subsystem>MTAOS</Subsystem>
-    <EFDB_Topic>MTAOS_logevent_ofcWarning</EFDB_Topic>
-    <Description>Warning of optical feedback control.</Description>
+    <EFDB_Topic>MTAOS_logevent_ofcDuration</EFDB_Topic>
+    <Description>Duration of time to calculate the degree of freedom.</Description>
     <item>
-      <EFDB_Name>timestamp</EFDB_Name>
-      <Description>Time at which an event is recorded.</Description>
-      <IDL_Type>double</IDL_Type>
+      <EFDB_Name>calcTime</EFDB_Name>
+      <Description>Average of calculation time.</Description>
```

Comment:
If this is a event instead of telemetry right now, we should publish the average calculation time or just the latest calculation time? Which one is more useful? I felt if this is a event now, I will prefer to use the latest calculation time.

Reply:
Updated to "Latest execution time"